docker-compose.yml

```diff
@@ -5,38 +5,66 @@ volumes:
   geth_data2:
   beacon_node_data:
   beacon_node_follower_data:
+  genesis_data:
+  lighthouse_shared_config:
+  lighthouse_data:
 
 services:
+  execution-bootnode:
+    build:
+      context: ./geth
+      dockerfile: Dockerfile.geth
+    command: /usr/local/bin/bootnode.sh
+    volumes:
+      - type: bind
+        source: ./geth/bootnode.sh
+        target: /usr/local/bin/bootnode.sh
+ 
   execution-node:
+    depends_on:
+      #- execution-bootnode
```

Comment:
I think we still need the `execution-node` to depend on the `execution-bootnode` right?

Reply:
I couldn't get geth to connect to the bootnode.
The original reason to have both EL clients peer was to synchronize blocks pre-merge. But now that we're post-merge at genesis, we no longer really need to do this.

It'll be nice to have the bootnode working though, so we can test the 4844 mempool across multiple EL implementations. But for now this is OK.